tests/test_warm_reboot.py

```diff
@@ -46,7 +46,7 @@ def swss_check_RestoreCount(dvs, state_db, restore_count):
             if fv[0] == "restore_count":
                 assert int(fv[1]) == restore_count[key] + 1
             elif fv[0] == "state":
-                assert fv[1] == "reconciled"
+                assert fv[1] == "reconciled" or fv[1] == "disabled"
```

Comment:
Do we still need to check for "disabled"? If not, please remove

Reply:
Yes, we still need "disabled" to handle the cold restart scenario, specially for docker restart